.github/workflows/ci.yml

```diff
@@ -61,16 +76,22 @@ jobs:
           target: ${{ env.NRF_TARGET }}
       - name: Install dependencies
         run: |
-          sudo apt-get update && sudo apt-get install libssl-dev
+          if [ "$RUNNER_OS" == "Linux" ]; then
+            sudo apt-get update && sudo apt-get install libssl-dev
```

Comment:
do we still need to install openssl after removing the `cargo-generate` installation?

Reply:
you're right, we don't– removed :)